packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -70,9 +75,71 @@ export interface AutocompleteApi<TItem> extends AutocompleteSetters<TItem> {
   destroy(): void;
 }
 
+export function getDropdownPositionStyle({
+  dropdownPlacement,
+  container,
+  inputWrapper,
+  environment = window,
+}: Partial<AutocompleteOptions<any>> & {
+  container: HTMLElement;
+  inputWrapper: HTMLElement;
+}) {
+  const containerRect = container.getBoundingClientRect();
+  const top = containerRect.top + containerRect.height;
+
+  switch (dropdownPlacement) {
+    case 'start': {
+      return {
+        top,
+        left: containerRect.left,
+      };
+    }
+
+    case 'end': {
+      return {
+        top,
+        right:
+          environment.document.documentElement.clientWidth -
+          (containerRect.left + containerRect.width),
+      };
+    }
+
+    case 'full-width': {
+      return {
+        top,
+        left: 0,
+        right: 0,
+        width: 'unset',
```

Comment:
you might want to fall back to `auto` for IE11 & early edge: https://caniuse.com/#feat=css-unset-value

Reply:
Ugh...

Reply:
I added in a TODO comment to move forward though.